private/aws-util-test/src/requests/test-http-handler.d.ts

```diff
@@ -0,0 +1,65 @@
+import { HttpHandler, HttpRequest, HttpResponse } from "@aws-sdk/protocol-http";
```

Comment:
this and the .js file should not need to be generated

Reply:
Those shouldn't have been committed.